### DISCUSSION TITLE
Odoo 16: links in addons path won't work

### DISCUSSION THREAD

**Discussion by yelizariev** [Tecnativa/doodba#467] [Doodba - The image]
Links to the modules won't work because of recent changes. See https://github.com/odoo/odoo/pull/82081

**Comment by joao-p-marques** [comment]
Mmm thanks for the heads up! I'm opening an issue to track this, ~~but AFAICS this will only land in v16, right?~~

**Reply by joao-p-marques** [reply]
Oh sorry, I didn't pay attention to the header :sweat_smile:
